src/pages/NewChatPage.js

```diff
@@ -129,6 +131,23 @@ class NewChatPage extends Component {
         return sections;
     }
 
+    /**
+     * Attempts to find a report in onyx with the provided list of participants
+     * @param {Array} newParticipantList
+     * @returns {Array|undefined}
+     */
+    getChatByParticipants(newParticipantList) {
+        const sortedNewParticipantList = newParticipantList.sort();
+        return _.find(this.props.reports, (report) => {
+            // Chat rooms don't have participants, so we can skip them
```

Comment:
Why do chat rooms not have participants?

Reply:
Just bad writing. Chat rooms _can_ not have participants and in that case we don't want to throw an error, we can just skip. I'll update it

Reply:
Updated